sechub-shared-kernel/src/test/java/com/mercedesbenz/sechub/sharedkernel/messaging/ListOfStringsMessageDataProviderTest.java

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.sharedkernel.messaging;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+class ListOfStringsMessageDataProviderTest {
+
+    private ListOfStringsMessageDataProvider providerToTest = new ListOfStringsMessageDataProvider();
+
+    @Test
+    void null_handled_correctly() {
+        /* execute */
+        List<String> profileIds = providerToTest.get(null);
+        String profileIdsAsString = providerToTest.getString(null);
+
+        /* test */
+        assertThat(profileIds).isNull();
+        assertThat(profileIdsAsString).isNull();
+    }
+
+    @Test
+    void empty_values_handled_correctly() {
```

Comment:
You check for null and you check for "[]"
what happens for providerToTest.get("") ?